docs/Off-line-Signatures-QR-Code.md

```diff
@@ -192,6 +211,55 @@ Generate the QR code from `offlineData`. Code example in Java:
             return "data:image/png;base64," + BaseEncoding.base64().encode(bytes);
 ```
 
-### 5. Verify ECDSA signature of offlineData
+The user can now scan the QR code via the mobile token app.
+
+The value of `nonce` must be stored somewhere - for example on a browser level, either as a hidden HTML form input field or as a React.js variable, or alternatively, it can be stored in the user session by the specific operation. The `nonce` value is required for later offline signature validation, see step 3.
```

Comment:
Not sure we need to mention React.js, using `JavaScript variable` is more generic...

Reply:
Good point.